From cf6b60c504d4be5e1df2b2745e55d677967831d0 Mon Sep 17 00:00:00 2001
From: Changhyeok Bae <changhyeok.bae@gmail.com>
Date: Tue, 9 Aug 2022 04:01:05 +0000
Subject: [PATCH] ipstats: Add param.h for musl

Fix build error for musl
| /usr/src/debug/iproute2/5.19.0-r0/iproute2-5.19.0/ip/ipstats.c:231: undefined reference to `MIN'

Signed-off-by: Changhyeok Bae <changhyeok.bae@gmail.com>
[ upstream status: cf6b60c504d4be5e1df2b2745e55d677967831d0 ]
---
 ip/ipstats.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/ip/ipstats.c b/ip/ipstats.c
index 5cdd15ae..1ac275bd 100644
--- a/ip/ipstats.c
+++ b/ip/ipstats.c
@@ -1,6 +1,7 @@
 // SPDX-License-Identifier: GPL-2.0+
 #include <assert.h>
 #include <errno.h>
+#include <sys/param.h>

 #include "list.h"
 #include "utils.h"
--
2.37.2

